### DISCUSSION TITLE
Multiple WANIPs

### DISCUSSION THREAD

**Discussion by shaoyihua** [shadowsocks/shadowsocks-rust#1470] [Q&A]
I have three external IPs, for example: 10.0.0.1, 10.0.0.2, 10.0.0.3
The main network card is 10.0.0.1, and the secondary IPs are 10.0.0.2 and 10.0.0.3. I can log in successfully using 10.0.0.2, but I still use 10.0.0.1 to access the Internet. How to configure 10.0.0.2 to log in, use 10.0. 0.2 to access the internet, use 10.0.0.3 to log in, use 10.0.0.3 to access the internet

**Comment by zonyitoo** [comment]
I don't think your question is related to this project.
